scripts/primaza.sh

```diff
@@ -56,7 +57,7 @@ function primazaUsage() {
 function build() {
   cmdExec "mvn clean install -DskipTests -Dquarkus.container-image.build=true \
      -Dquarkus.container-image.push=true \
-     -Dquarkus.container-image.registry=127.0.0.1:5000 \
+     -Dquarkus.container-image.registry=127.0.0.1:${REGISTRY_PORT} \
```

Comment:
REGISTRY hostname should not be hard coded too `127.0.0.1` !

Reply:
It's ok. Look the last change and my previous comment